packages/ui/src/walkthrough-card/WalkthroughCard.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+import { useRouter } from "next/navigation";
+
+import { WalkthroughInfo } from "@repo/data/useWalkthroughData";
+
+import "./WalkthroughCard.css";
+
+export interface WalkthroughCardProps {
+  id: string;
+  data: WalkthroughInfo;
+  href?: string;
+}
+
+export default function WalkthroughCard({
+  id,
+  data,
+  href,
+}: WalkthroughCardProps) {
+  const router = useRouter();
+
+  const handleClick = () => {
+    if (href) router.push(href);
+  };
+
+  const canDisplayCard =
+    data && data.title && data.subtitle && data.description;
+
+  if (!canDisplayCard) return null;
+
+  return (
+    <a
```

Comment:
Sorry, I should have said this different the first time. This should use either the react-aria-components Link or the next/link to take advantage of client side routing. This way we can just add an href to it and it'll also be accessible. We don't need to handle any clicking or use the router here.